aws/table_aws_ses_email_identity.go

```diff
@@ -0,0 +1,195 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ses"
+	"github.com/turbot/go-kit/types"
+	"github.com/turbot/steampipe-plugin-sdk/v3/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin/transform"
+)
+
+func tableAwsSESEmailIdentity(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_ses_email_identity",
+		Description: "AWS SES Email Identity",
+		List: &plugin.ListConfig{
+			Hydrate: listSESEmailIdentities,
+		},
+		GetMatrixItem: BuildRegionList,
+		Columns: awsRegionalColumns([]*plugin.Column{
+			{
+				Name:        "name",
+				Description: "The email identity.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromValue(),
```

Comment:
Do we need FromValue() here?

Reply:
Yes, since the value returned from the hydrate call is only a string - "identity-name".